includes/usermaven-settings-form.php

```diff
@@ -11,13 +11,13 @@ function usermaven_activation_form() {
     $cookie_less_tracking = isset( $_POST['cookie_less_tracking'] ) ? true : false;
     $embed_dashboard = isset( $_POST['embed_dashboard'] ) ? true : false;
 
-    $api_key = esc_attr($_POST['api_key']);
+    $api_key = sanitize_text_field($_POST['api_key']);
```

Comment:
I think we should rename these functions to usermaven_sanitize_text_field. and similarly other functions.